src/ik_evolution_2.cpp

```diff
@@ -237,24 +238,27 @@ template <int memetic> struct IKEvolution2 : IKBase
     */
 
     // aligned_vector<double> rmask;
-
+#pragma GCC push_options
+#pragma GCC optimize ("unroll-loops")
     // create offspring and mutate
     __attribute__((hot)) __attribute__((noinline))
+    __attribute__((optimize("unroll-loops")))
```

Comment:
Why did you add this?

Reply:
There was a `#pragma unroll` below it that was being ignored by gcc

Reply:
So we added this attribute to silence a warning? Can we remove the pragmas that caused the warning then? Without benchmarks, we have no evidence to suggest that our manual use of attributes can beat what the compiler is already doing.

Reply:
Removed in [e9c01ee](https://github.com/PickNikRobotics/bio_ik/pull/1/commits/e9c01ee4ef00ced5d79bb788ab01a067e93e922c)